src/Python/sirf/contrib/NEMA/generate_nema_rois.py

```diff
@@ -199,8 +203,14 @@ def generate_nema_rois(data_output_path, sino_file, image_size_xy):
     np.set_printoptions(precision=3,suppress=True)
     TM = algo.get_transformation_matrix_forward()
     print(TM.as_array())
+    return TM, reg_image, sphere_nii
 
-    #once we have the registration matrix we can the apply it to the sphere generation?
+def generate_nema_rois(recon_image):
+    
+    construct_NEMA_spheres_and_save(acq_data,image_size)
```

Comment:
where does `acq_data` and `image_size` come from? Probably global variables only defined when using as a script. Better to pass recon_image (as a template), and use `image.get_uniform_copy(0)` in `construct_NEMA_spheres_and_save`.